src/sage/matroids/dual_matroid.py

```diff
@@ -526,3 +502,64 @@ def __reduce__(self):
         data = (self._matroid, self.get_custom_name())
         version = 0
         return sage.matroids.unpickling.unpickle_dual_matroid, (version, data)
+
+    def relabel(self, mapping):
+        r"""
+        Return an isomorphic matroid with relabeled groundset.
+
+        The output is obtained by relabeling each element ``e`` by
+        ``mapping[e]``, where ``mapping`` is a given injective map. If
+        ``e not in mapping`` then the identity map is assumed.
+
+        INPUT:
+
+        - ``mapping`` -- a python object such that `mapping[e]` is the new
+          label of `e`
```

Comment:
```suggestion
        - ``mapping`` -- a Python object such that ``mapping[e]`` is the new
          label of ``e``
```

Reply:
I think that `e` is better than ``e`` (change it elsewhere too).